add_connector.md

```diff
@@ -9,6 +9,13 @@ This is a guide to contributing new connector to Router. This guide includes ins
 - Understanding of the Connector APIs which you wish to integrate with Router
 - Setup of Router repository and running it on local
 - Access to API credentials for testing the Connector API (you can quickly sign up for sandbox/uat credentials by visiting the website of the connector you wish to integrate)
+-  Ensure that you have Rust installed. We recommend using the nightly version. Install it using `rustup`:
```

Comment:
We only need the nightly toolchain for formatting, trying to compile our project on the nightly toolchain can cause the compiler to panic. Let's suggest using the stable toolchain itself.